docs/tutorials/pmac.rst

```diff
@@ -70,7 +70,7 @@ Define the Process Definition
 
 Let's make a Process Definition in ``etc/malcolm/BLxxI-ML-MALC-01.yaml``::
 
-    #!/dls_sw/prod/common/python/RHEL7-x86_64/pymalcolm/4-x/prefix/bin/imalcolm
+    #!/dls_sw/prod/tools/RHEL7-x86_64/defaults/bin/imalcolm
```

Comment:
I'd prefer to keep the version number in there in examples too, to stop people copy and pasting the wrong thing...

Reply:
I have now pointed it at release 4.4 in the example.